src/bluetoothdevice.cpp

```diff
@@ -278,11 +286,40 @@ QStringList bluetoothdevice::metrics() {
 
 resistance_t bluetoothdevice::maxResistance() { return 100; }
 
+bool bluetoothdevice::updateLockscreenHeartRate() {
+
+#ifdef Q_OS_IOS
+#ifndef IO_UNDER_QT
+    lockscreen h;
+    long appleWatchHeartRate = h.heartRate();
+    this->Heart = appleWatchHeartRate;
+    qDebug() << "Current Heart from Apple Watch: " << QString::number(appleWatchHeartRate);
```

Comment:
Changed from using the subclass' debug signal because that signal is not defined at this level.